### PR TITLE
Chore: use automatic instance manager

### PR DESCRIPTION
**Pull request by ryantxu**
update to the latest SDK and use the automatic instance management

### CODE REVIEW

File: pkg/server/server.go [hunk 1]
```diff
@@ -15,26 +16,26 @@ import (
 )
 
 type Server struct {
-	Datasource Datasource
+	Datasource    Datasource
+	channelPrefix string
+	closeCh       chan struct{}
+	queryMux      *datasource.QueryTypeMux
```
Comment (ryantxu) [inline pkg/server/server.go:22]: Not used yet -- but these will be helpful when we support streaming

File: pkg/server/server.go [hunk 2]
```diff
@@ -15,26 +16,26 @@ import (
 )
 
 type Server struct {
-	Datasource Datasource
+	Datasource    Datasource
+	channelPrefix string
+	closeCh       chan struct{}
```
Comment (ryantxu) [inline pkg/server/server.go:21]: Not used yet -- but these will be helpful when we support streaming

File: pkg/server/server.go (hunk 1 above)
Comment (ryantxu) [inline pkg/server/server.go:22]: rather than build the map on every request, this saves builds it once on startup

**Comment by ryantxu** [comment]
@bymattoa -- no rush on this.  It should not change any behavior, only shifts things to use a shared wrapper for instance management.  This at least removes one extra interface abstraction :man_shrugging:

**Review by bymattoa — APPROVED** [review]

**Pull request merged**